src/test/java/com/wcc/platform/factories/SetupMentorshipFactories.java

```diff
@@ -139,6 +144,37 @@ private static CommonSection createCommonSectionOnlyLinkTest() {
     return new CommonSection(null, null, null, createLinkTest(), null, null);
   }
 
+  /** Test factory for Study Group Page. */
+  public static MentorshipStudyGroupsPage createMentorshipStudyGroupPageTest(
```

Comment:
Could you review this file, Sonar is complaining of duplicated import. I think if you just change some spaces and save, with the "Actions on Save" it runs Optimize imports and it is going to fix that for you.

Reply:
I will merge like this and we can fix in another PR.